### PR TITLE
[AF-549-elastic]: Upgraded Elasticserch version to 5.6.1.

### PR DESCRIPTION
**Pull request by adrielparedes**
@dgutierr @romartin I had to put this PR with the others that involve changing ES and Lucene version

I think this PR is old now with this project move, should we close it?
 https://github.com/dashbuilder/dashbuilder/pull/390

### CODE REVIEW

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by adrielparedes** [comment]
jenkins retest this

File: kie-soup-dataset/kie-soup-dataset-elasticsearch/pom.xml [hunk 1]
```diff
@@ -35,11 +35,47 @@
...
       </plugin>
-      
+
     </plugins>
-    
+
   </build>
 
+
+  <dependencyManagement>
```
Comment (mbarkley) [inline kie-soup-dataset/kie-soup-dataset-elasticsearch/pom.xml]: Can you please move these to the dependencyManagement section in the kie-soup-parent POM with all the other third-party dependencies? Also please use properties for all the versions, following the convention of other property names (i.e. `version.org.apache.log4j`).

File: kie-soup-dataset/kie-soup-dataset-elasticsearch/pom.xml [hunk 2]
```diff
@@ -1,4 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright ${year} Red Hat, Inc. and/or its affiliates.
```
Comment (mbarkley) [inline kie-soup-dataset/kie-soup-dataset-elasticsearch/pom.xml]: Just noticed the year needs to be set.

Comment (adrielparedes) [reply kie-soup-dataset/kie-soup-dataset-elasticsearch/pom.xml]: @mbarkley thanks, I thought that I've fixed those wrong replacements but it seems not

**Pull request merged**